ykrt/src/jitstats.rs

```diff
@@ -17,7 +14,14 @@ use std::{
 };
 use strum::{Display, EnumCount, EnumIter, IntoEnumIterator};
 
+/// Record yk statistics if enabled. In non-testing mode this is only enabled if the end user
+/// defines the global variable `YKD_JITSTATS`. In testing mode this is always enabled, with output
+/// being sent to `stderr`.
```

Comment:
What's the justification for "always enabled by default in testing?". I'd expect it off by default like all the other things.

Reply:
Hmm, that's a good question. You're right, it probably shouldn't be always on.

If it's OK, I'll fix that in the next PR.

Reply:
Please do :)

Reply:
In the next PR, can it also be a `0/1`-valued variable for consistency?

Reply:
That wouldn't make sense -- you have to give a filename (or `-` for stderr) as an argument.

Reply:
ah yeah, my bad.